test-configs.yaml

```diff
@@ -1635,16 +1675,56 @@ test_configs:
       - baseline
       - kselftest
 
+  - device_type: imx6qp-sabresd
+    test_plans:
+      - baseline
+      - kselftest
+
+  - device_type: imx6sll-evk
+    test_plans:
+      - baseline
+      - kselftest
+
+  - device_type: imx6sx-sdb
+    test_plans:
+      - baseline
+      - kselftest
+
+  - device_type: imx6ul-14x14-evk
+    test_plans:
+      - baseline
+      - kselftest
+
+  - device_type: imx6ull-14x14-evk
```

Comment:
Baseline OK: https://staging.kernelci.org/test/plan/id/5efdb0a6532816a47348b917/
Upstream in LAVA: https://git.lavasoftware.org/lava/lava/-/blob/master/etc/dispatcher-config/device-types/imx6ull-14x14-evk.jinja2